### PR TITLE
add benchmark collection references

### PR DESCRIPTION
**Pull request by plakrisenko**

### CODE REVIEW

**Comment by plakrisenko** [comment]
I guess the benchmark collection wasn't really used in StaedterSch2021?

File: src/benchmark_refs.bib [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@article{RaimundezFed2023,
+	title = {Posterior marginalization accelerates Bayesian inference for dynamical models of biological processes},
+	journal = {iScience},
+	volume = {26},
+	number = {11},
+	pages = {108083},
+	year = {2023},
+	issn = {2589-0042},
+	doi = {https://doi.org/10.1016/j.isci.2023.108083},
```
Comment (dweindl) [inline src/benchmark_refs.bib:9]:
```suggestion
	doi = {10.1016/j.isci.2023.108083},
```

**Comment by dweindl** [comment]
> I guess the benchmark collection wasn't really used in StaedterSch2021?

Yes, only biomodels / jws afaik.

**Review by dweindl — APPROVED** [review]
Thanks. Please add a note to the README that there is this list.

**Pull request merged**